src/cmd/sql_cmd.h

```diff
@@ -818,8 +818,7 @@ base::Status HandleDeploy(const hybridse::node::DeployPlanNode* deploy_node) {
 }
 
 void SetVariable(const std::string& key, const hybridse::node::ConstNode* value) {
-    std::string lower_key = key;
-    boost::to_lower(lower_key);
+    auto lower_key = boost::to_lower_copy(key);
```

Comment:
`boost::to_lower_copy` is better?

Reply:
Not a big difference.
Just thought two steps bit slower than the new one, intuitively.